### ISSUE TITLE
Not working in WG protected area

### ISSUE THREAD

**Issue by ebins** [croxis/Lift#133]
Lift is simply not working in a WorldGuard protected area unless that person has OP or is a member of said region. It works perfectly in non WorldGuard protected areas. I've tried enabling the use flag and the interact flag, to no avail. I want to use this in my world's Spawn.

I'm running Spigot 1.8.8 and WorldGuard version 6.1 I'm also running Java 8. 

No error messages are being generated. I just won't work.

**Comment by croxis** [comment]
I will look into it. I might not get to it today (got to finish lesson planning for the week) but I'll try and get to it this week.

**Comment by ebins** [comment]
Ok, thanks :)

**Comment by ebins** [comment]
I have an update for you. 

I'm an idiot. 

I had Spawn Protection turned on in server.properties, and this is what was preventing the lift to work, not worldguard. I had no idea that this would be a problem, and it wasn't until I started researching an issue with another plugin that I came across the solution. After removing spawn protection, I re-tested the elevators and they work fine. 

I'm so very sorry, and I really hope I haven't wasted much of your time trying to debug code that works perfectly. :(

Ebins
